src/framework/theme/components/layout/_layout.component.theme.scss

```diff
@@ -165,16 +167,20 @@
           box-shadow: nb-theme(footer-shadow);
 
           nav {
-            padding: nb-theme(footer-padding);
-            background: nb-theme(footer-bg);
-            color: nb-theme(footer-fg);
+            background-color: nb-theme(footer-background-color);
             border-top: 1px solid nb-theme(footer-separator);
```

Comment:
there is no such variable as `footer-separator`

Reply:
Fixed in 28f6475fc70686643c12108f49593eeeb80520c9